urban-os/values.yaml

```diff
@@ -79,3 +88,282 @@ kubernetes-data-platform:
+              expr: probe_duration_seconds{instance=~".*(/api/v1/predictions)"} > 3
+              for: 1m
+              labels:
+                severity: error
+              annotations:
+                description: "{{ $labels.instance }} of job {{ $labels.job }} is taking more than 3 seconds."
+                summary: "Instance {{ $labels.instance }} is taking longer response time"
+        - name: K8S_Nodes
+          rules:
+            - alert: LowMemory
+              expr: (node_memory_MemAvailable_bytes / node_memory_MemTotal_bytes * 100) < 20
+              for: 5m
+              labels:
+                severity: warning
+              annotations:
+                description: "{{ $labels.instance }} has {{ $value }} percent memory left."
+                summary: "Low Memory on Instance {{ $labels.instance }}"
+            - alert: LowDisk
+              expr: (node_filesystem_avail_bytes{device=~"/dev/.*"} / node_filesystem_size_bytes{device=~"/dev/.*"} * 100) < 15
+              for: 5m
+              labels:
+                severity: warning
+              annotations:
+                description: "{{ $labels.instance }} has {{ $value }} percent disk left."
+                summary: "Low Disk on Instance {{ $labels.instance }}"
+            - alert: LowClusterCPU
+              expr: (cluster:capacity_cpu:sum - cluster:guarantees_cpu:sum) < 1
+              labels:
+                severity: warning
+              annotations:
+                description: "Kubernetes cluster has {{ $value }} cores left. New deployments and cron jobs may fail to launch."
+                summary: "Kubernetes cluster low on CPU cores"
+            - alert: LowClusterMemory
+              expr: (cluster:capacity_memory_bytes:sum - cluster:guarantees_memory_bytes:sum) < 1000000000 #1GB
+              labels:
+                severity: warning
+              annotations:
+                description: "Kubernetes cluster has {{ $value | humanize }} memory left. New deployments and cron jobs may fail to launch."
+                summary: "Kubernetes cluster has less than {{ 1000000000.0 | humanize }} memory available"
+            - alert: NotReady
+              annotations:
+                description: "{{ $labels.node }} is not ready"
+                summary: "Status not Ready on Node {{ $labels.node }}"
+              expr: kube_node_status_condition{condition="Ready", status="true"} != 1
+              labels:
+                severity: error
+        - name: tracer_alerts
+          rules:
+            - alert: MessageThroughput
+              expr: sum(rate(kafka_topic_partition_current_offset{topic=~"raw-00000000-7e77-4b1c-92a4-36e09db56173|transformed-00000000-7e77-4b1c-92a4-36e09db56173|streaming-persisted|scos__sample_streaming_dataset"}[5m])) by (topic) == 0
+              labels:
+                severity: error
+              annotations:
+                summary: "{{ $labels.topic }} has no input for at least 5 minutes"
```

Comment:
The tracer alert mentions a dataset with an id specific to the SCOS instance. This should probably be pulled out to the SCOS repo.

Reply:
Moved